server/src/get-options.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 
-source /usr/share/bash-completion/bash_completion
+DATADIR="$(pkg-config --variable=datadir bash-completion)"
```

Comment:
How does the script handle if the user doesn't have the completion installed?

```
~ pkg-config --variable=datadir bash-completion

package bash-completion was not found in the pkg-config search path.
Perhaps you should add the directory containing `bash-completion.pc'
to the PKG_CONFIG_PATH environment variable
No package 'bash-completion' found

~ echo $?
1
```

Reply:
At least on OS X the suggested solution doesn't work. Not sure if the solution works on most unix distros?

Reply:
It should work if the `bash-completion` package is installed and it provides a `bash-completion.pc` file.